ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -1057,19 +1093,8 @@ private class JdbcLedgerDao(
     SQL(
       // the distinct keyword is required, because a single contract can be visible by 2 parties,
```

Comment:
This comment should probably move to where `SQL_SELECT_ACTIVE_CONTRACTS` is defined

Reply:
👍